### DISCUSSION TITLE
Fatal Error Tweakscale @Lisias

### DISCUSSION THREAD

**Discussion by CadenMcAllister** [TweakScale/TweakScale#342] [Support] [answered]
@Lisias

![Screenshot 2024-11-26 120028](https://github.com/user-attachments/assets/55019cc1-82d5-498c-a842-2426b4dcb61c)

I am receiving this issue after running ksp.
Here is my log 
[KSP.log](https://github.com/user-attachments/files/17924027/KSP.log)

**Comment by CadenMcAllister** [comment]
@Lisias

**Comment by Lisias** [comment, marked as answer]
Hi, @CadenMcAllister !

You (or something else) installed an alien fork over TweakScale, or viceversa.

I suggest you do manually **COMPLETELY REMOVE** the following Directories:

* `GameData/TSSafetyNet`
* `GameData/TweakScale`

And then reinstall from scratch TweakScale.

**Reply by CadenMcAllister** [reply]
I'll try that. will let you know if it works
